lib/plugins/aws/deploy/compile/events/apiGateway/index.js

```diff
@@ -24,37 +27,35 @@ class AwsCompileApigEvents {
       compileRestApi,
       compileApiKeys,
       compileResources,
+      configurePreflightMethods,
```

Comment:
I think I will rename this to `compileCors` - sorry leftover from pulling in other changes.